include/oneapi/dpl/pstl/hetero/algorithm_ranges_impl_hetero.h

```diff
@@ -756,15 +942,30 @@ __pattern_merge(__hetero_tag<_BackendTag> __tag, _ExecutionPolicy&& __exec, _Ran
 }
 
 #if _ONEDPL_CPP20_RANGES_PRESENT
+
+template <typename _Comp, typename _Proj1, typename _Proj2>
+struct __pattern_merge_comp
```

Comment:
We already defined a type with the same functionality above. And we can re-use one.

Reply:
Fixed